mon-entreprise/scripts/fetch-stats.js

```diff
@@ -247,7 +247,14 @@ async function main() {
 		const visitesMois = await fetchMonthlyVisits()
 		const satisfaction = uniformiseData(
 			flattenPage(await fetchApi(buildSatisfactionQuery()))
-		)
+		).map((page) => {
+			// eslint-disable-next-line no-unused-vars
+			const { date, ...satisfactionPage } = {
```

Comment:
Why not use `delete` on `date`?

Reply:
Je préfère éviter l'impératif quand possible, et ça me semble en ligne avec le style du projet jusqu'aujourd'hui.

Selon toi, c'est non-standard d'écrire comme ça?

Côté eslint, il y a `ignoreRestSiblings: true` qui peut être utilisé dans la règle `no-unused-vars` pour normaliser ça dans le projet, et éviter le `eslint-disable-next` plus tard.

Reply:
Je me souviens plus si `eslint` comprends les variables préfixées avec un `_` comme étant implicitement non utilisées ? Si c'est le cas il y a aussi l'option :

```suggestion
			const { date: _date, ...satisfactionPage } = {
```

Reply:
Tout me va, mais ça ne me choque pas d'utiliser `delete` sur un objet cloné. Vu qu'on essaye de limiter le recours à ramda, c'est pas choquant de se retrouver avec du code un peu plus impératif, tant que c'est de manière très localisé en respectant des principes de non mutation des structures de données en argument d'une fonction.